packages/slider/src/uxp-slider.css

```diff
@@ -0,0 +1,171 @@
+        var(--spectrum-slider-label-top-to-text)
+    );
+}
+
+#label {
+    padding-left: 0;
+    padding-top: calc(
+        var(--mod-slider-font-size, var(--spectrum-slider-font-size)) / 3
+    );
+}
+
+#value {
+    margin-left: var(
+        --mod-slider-label-margin-start,
+        var(--spectrum-slider-label-margin-start)
+    );
+    padding-right: 0;
+}
+
+#controls {
+    height: var(
+        --mod-slider-control-height,
+        var(--spectrum-slider-control-height)
+    );
+    width: calc(
+        100% -
+            var(
+                --mod-slider-controls-margin,
+                var(--spectrum-slider-controls-margin)
+            ) * 2
+    );
+    margin-left: var(
+        --mod-slider-controls-margin,
+        var(--spectrum-slider-controls-margin)
+    );
+}
+
+#fill,
+.track {
+    height: var(
+        --mod-slider-track-fill-thickness,
+        var(--spectrum-slider-track-fill-thickness)
+    );
+    top: calc(
+        var(--mod-slider-control-height, var(--spectrum-slider-control-height)) /
+            2 -
+            var(
+                --mod-slider-track-fill-thickness,
+                var(--spectrum-slider-track-fill-thickness)
+            ) / 2
+    );
+    left: 0 auto;
+    margin-left: var(
+        --mod-slider-track-margin-offset,
+        var(--spectrum-slider-track-margin-offset)
+    );
+    padding-top: 0;
+    padding-bottom: 0;
+    padding-right: var(
+        --mod-slider-handle-gap,
+        var(--spectrum-slider-handle-gap)
+    );
+    padding-left: 0;
+}
+
+#fill:before,
+.track:before {
+    height: 100%;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+    border-top-right-radius: 0;
+    border-top-left-radius: 0;
+}
+
+.handle {
+    height: var(--mod-slider-handle-size, var(--spectrum-slider-handle-size));
+    width: var(--mod-slider-handle-size, var(--spectrum-slider-handle-size));
+    top: calc(
+        var(--mod-slider-control-height, var(--spectrum-slider-control-height)) /
+            2 -
+            var(--mod-slider-handle-size, var(--spectrum-slider-handle-size)) /
+            2
+    );
+    left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+    margin-left: calc(
+        var(--mod-slider-handle-size, var(--spectrum-slider-handle-size)) / -2
+    );
+    margin-right: calc(
+        var(--mod-slider-handle-size, var(--spectrum-slider-handle-size)) / -2
+    );
+}
+
+.handle:before {
+    height: var(--mod-slider-handle-size, var(--spectrum-slider-handle-size));
+    width: var(--mod-slider-handle-size, var(--spectrum-slider-handle-size));
+}
+
+.input {
+    height: var(--mod-slider-handle-size, var(--spectrum-slider-handle-size));
+    width: var(--mod-slider-handle-size, var(--spectrum-slider-handle-size));
+    top: var(
+        --mod-slider-input-top-size,
+        var(--spectrum-slider-input-top-size)
+    );
+    left: var(--mod-slider-input-left, var(--spectrum-slider-input-left));
+    opacity: 0;
```

Comment:
Have we verified the events in Photoshop after setting opacity to 0?

Reply:
Yes i have verified this and events are working fine in PS.

I had to do this to overcome candle stick like handle coming over the circular handle in PS when sp-slider is rendered.

Apart from this we have a known issue in sp-slider related to tab-navigation which i have created a JIRA - https://jira.corp.adobe.com/browse/UXP-22866 to be taken in UXP-8.2